src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../appTypeDefs';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type MoviesListProps = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<MoviesListProps> = (
+  { movies },
+) => {
+  return (
```

Comment:
you can opt in for using an implicit return whenever there's no actual function body
```suggestion
) => (
```

Reply:
Okay thanx